Updated Figma to BabylonJS community extension docs.

jamessimo opened the pull request:
Added more information, more screenshots and covered new features added in the plugins v16 launch of Figma to BabylonJS

Vinc3r commented on content/resources/Library/Running_a_local_webserver_for_BabylonJs.md:
````diff
@@ -0,0 +1,75 @@
+```
+
+
+**On windows**
````
Rather than bold, should'nt be better to use title `###`? (same for **other OS**)

deltakosh replied:
yes probably!

aWeirdo replied:
Hi guys!
I wanted to keep it out of the table of contents and hopefully avoid any confusion, as the OS sub-sections are not complete guides and the "Sample" steps before you reach the sub-sections are the same for all.

so basicly, i didn't want someone to go directly from "Nodejs & NPM" and to "Other OS", get confused of the missing context and start making a forum question 😅 

That was the thought behind it, but I won't get mad if you prefer it the other way 😄

deltakosh replied:
Well that makes sense so fine for me :)

Vinc3r replied:
'Understood, I lower my weapons :D

jamessimo committed:
Updated Figma to BJS docs

Added more information, more screenshots and covered new features added in the plugins v16 launch.

bjsplat commented:
snapshot for this PR available at:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/DOCS/refs/pull/1419/merge
